constraint/pkg/client/client.go

```diff
@@ -553,9 +555,14 @@ func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.Qu
 			continue
 		}
 
+		// edge case: .Milliseconds() can be rounded to 0 so take the nanos and convert to millis
+		resp.EvaluationLatency = float64(time.Since(reviewStart).Nanoseconds()) / 1000000
```

Comment:
Notice we are just collecting this information. Consumers of the library will log/ emit/ analyze this as needed.